aplib.net-demo/Assets/Scripts/Doors/DoorCollision.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// This class handles collisions with the player and makes sure that the parent object (the door) disappears/opens when the players is in range and the prerequisites are met (the right key, or in this case the right player ID)
+/// Set the ID in the editor for the collider
+/// </summary>
+public class DoorCollision : MonoBehaviour
+{
+    public int doorID;
```

Comment:
```suggestion
    public int doorId;
```
I explained why the d should be lowercase later on.